source/iterator/cdc.go

```diff
@@ -0,0 +1,370 @@
+// Copyright © 2022 Meroxa, Inc & Yalantis.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package iterator
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"sync"
+	"time"
+
+	sdk "github.com/conduitio/conduit-connector-sdk"
+	"github.com/huandu/go-sqlbuilder"
+	"github.com/jmoiron/sqlx"
+
+	"github.com/conduitio-labs/conduit-connector-db2/coltypes"
+	"github.com/conduitio-labs/conduit-connector-db2/source/position"
+)
+
+const (
+	waitingTimeoutSec            = 20
+	clearTrackingTableTimeoutSec = 5
+)
```

Comment:
As I remember we have discussed that less config variables we have  that is better for user.  But as for me we can add these fields to config. Only with some validations. Because if user sets  big values it is can be problem with performance